src/store/modules/wallet.js

```diff
@@ -191,19 +207,23 @@ export default {
     },
     async updateUTXOFromScriptHash ({ commit, rootGetters, getters }, scriptHash) {
       let client = rootGetters['electrumHandler/getClient']
-      var elOutputs = await client.methods.blockchain_scripthash_listunspent(scriptHash)
-      let { address } = getters['getAddressByElectrumScriptHash'](scriptHash)
-      let outputs = elOutputs.map(elOutput => {
-        let output = {
-          txId: elOutput.tx_hash,
-          outputIndex: elOutput.tx_pos,
-          satoshis: elOutput.value,
-          type: 'p2pkh',
-          address
-        }
-        return output
-      })
-      commit('refreshUTXOsByAddr', { addr: address, outputs })
+      try {
+        var elOutputs = await client.methods.blockchain_scripthash_listunspent(scriptHash)
+        let { address } = getters['getAddressByElectrumScriptHash'](scriptHash)
+        let outputs = elOutputs.map(elOutput => {
+          let output = {
+            txId: elOutput.tx_hash,
+            outputIndex: elOutput.tx_pos,
+            satoshis: elOutput.value,
+            type: 'p2pkh',
+            address
+          }
+          return output
+        })
+        commit('refreshUTXOsByAddr', { addr: address, outputs })
+      } catch (err) {
+        console.error('error deserializing utxo address', err, scriptHash)
```

Comment:
Perhaps it'd be better to simply add information to the errors, log them and then rethrow them, so they can be handled elsewhere?

This applies all the added try catch blocks.

Reply:
Depends on the case.  I think that salvaging all the salvageable UTXOs and logging the ones that aren't, is probably correct here.  Re-throwing would stop any further import.  It would be good to flag it to the end user though possibly.